javascript/javascript3/week1/index.js

```diff
@@ -25,4 +25,69 @@
   // Along with above URL one need to use parameters like cityname, zip code, geolocation and api key as well
   // one example of using this api with different parameters is following 
   //'https://api.openweathermap.org/data/2.5/weather?q=London&units=metric&appid=2e67673623b242fce63358a81c88036e'
-  
\ No newline at end of file
+  
+// !--Weather app--!
+
+const cityName = document.getElementById("heading")
+      const temperature = document.getElementById("temppara")
+      const feeltemp = document.getElementById("feelslike")
+      const wind = document.getElementById("cl-1")
+      const description = document.getElementById("cl-2")
+      const riseTime = document.getElementById("cl-3")
+      const sunTime = document.getElementById("cl-4")
+      
+      const myButton = document.getElementById("location")
+      function getWeather(myData){
+        
+        
+        const city =  myData.name
+        const mytemp = myData.main.temp
+        const feeling = myData.main.feels_like
+        const windSpeed = myData.wind.speed
+        const weatherDes = myData.weather[0].description
+        const sunRise = myData.sys.sunrise
+        const sunSet = myData.sys.sunset
+        const date1 = new Date(sunRise * 1000)
+        const hours1 = date1.getHours()
+        const minutes1 = date1.getMinutes()
+        const date2 = new Date(sunSet * 1000)
+        const hours2 = date2.getHours()
+        const minutes2 = date2.getMinutes()
```

Comment:
You could use object destructuring here rather than initializing these variables.